bolt-modules/boltlib/lib/puppet/functions/catch_errors.rb

```diff
@@ -8,8 +8,7 @@
 Puppet::Functions.create_function(:catch_errors) do
   # @param error_types An array of error types to catch
   # @param block The block of steps to catch errors on
-  # @return Undef If an error is raised in the block then the error will be
-  # returned, otherwise the result will be returned
+  # @return if an error is raised in the block then the error will be returned, otherwise the result will be returned
```

Comment:
Thats good to know. Looks like if we wanted/needed longer than 120 line length we just need some special indentation like https://github.com/puppetlabs/bolt/blob/de5fbd26dcf0be04e8e6c31e71f4201bb8027cf8/bolt-modules/boltlib/lib/puppet/functions/run_script.rb#L9-L10